### PR TITLE
Lighting_Engine: Added Create Methods for Lux Results, LuxSeries Results, and Mesh Results

### LINKED ISSUE #2779: Lighting_Engine: MeshResult Create Method(s) to enable workflows for results without software pull
#### Description:
Create methods to create Lighting-based mesh results per provided inputs that can be used downstream for analytical mesh workflows.

### PR DESCRIPTION
**Pull request by enarhi**
### NOTE: Depends on 
https://github.com/BHoM/BHoM/pull/1391

   
### Issues addressed by this PR

Closes #2779 

Create methods added to enable creation of require Lighting results objects to allow for MeshResult-based workflows for Lighting where an external toolkit is not used to generate them.

### Test files
https://burohappold.sharepoint.com/:u:/r/sites/BHoM/02_Current/12_Scripts/02_Pull%20Request/BHoM/BHoM_Engine/Lighting_Engine/%232836-LightingMeshAndSeriesResults.gh?csf=1&web=1&e=W04L1Z

### CODE REVIEW

**Comment by enarhi** [comment]
@BHoMBot check compliance

File: Lighting_Engine/Create/Lux.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+using BH.Engine.Spatial;
+
+namespace BH.Engine.Lighting
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+        public static MeshElementResult Lux(List<double> lux, IComparable objectId, IComparable nodeId, IComparable meshFaceId, IComparable resultCase, MeshResultSmoothingType smoothing)
```
[the remark this answers is not in the document]
Comment (enarhi) [reply Lighting_Engine/Create/Lux.cs:47]: Smoothing removed.

**Review by IsakNaslundBh — COMMENTED** [review]
Descriptions required. Besides that, this looks good to go.

**Comment by enarhi** [comment]
@BHoMBot check compliance

**Review by IsakNaslundBh — APPROVED** [review]
Works as intended

**Comment by IsakNaslundBh** [comment]
@BHoMBot check required

**Comment by IsakNaslundBh** [comment]
@BHoMBot check core
@BHoMBot check null-handling

**Comment by IsakNaslundBh** [comment]
@BHoMBot check installer
@BHoMBot check versioning

**Comment by IsakNaslundBh** [comment]
@BHoMBot check versioning

**Comment by enarhi** [comment]
@IsakNaslundBh can we do final checks and get this one merged?

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Pull request merged**